java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -431,6 +497,34 @@ void executeInstruction(Instruction instruction) {
     checkerDispatcher.executeCheckPostStatement(instruction);
   }
 
+  private static ProgramState stackValues(ProgramState.Pop pop, int... values) {
+    ProgramState ps = pop.state;
+    for (int value : values) {
+      ps = ps.stackValue(pop.values.get(value));
+    }
+    return ps;
+  }
+
+  private ProgramState setDoubleOrLong(SymbolicValue sv) {
```

Comment:
I think I would prefer to have this one removed, it is only used once and add one indirection for not much gain.